operations/validate_csm_health.md

```diff
@@ -15,6 +15,16 @@ The areas should be tested in the order they are listed on this page. Errors in
 Each section of this health check document provides links to relevant troubleshooting procedures. If additional help is needed, see
 [CSM Troubleshooting Information](../troubleshooting/README.md).
 
+Before running health checks, restart `goss-servers` on all NCNs to ensure the correct tests are run on each node. This is necessary because of a timing issue on the nodes that has been fixed in CSM 1.6.1.
```

Comment:
Looking at the health checks on this page, I think we should move this to the beginning of the Platform health checks section. This is for two reasons:

1. Those are the only Goss tests called on this page
2. Other pages link directly to that item, in which case they could miss this blurb at the top.

Reply:
Makes sense. I have moved it.